intl/msg_hash_us.h

```diff
@@ -8755,6 +8755,14 @@ MSG_HASH(
     MENU_ENUM_SUBLABEL_OZONE_COLLAPSE_SIDEBAR,
     "Have the left sidebar always collapsed."
     )
+MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_OZONE_TRUNCATE_PLAYLIST_NAME,
+    "Truncate Playlist names"
+    )
+MSG_HASH(
+    MENU_ENUM_SUBLABEL_OZONE_TRUNCATE_PLAYLIST_NAME,
+    "Remove everything before the hyphen and the subsequent space in Playlist names. Changes require a restart to take effect."
```

Comment:
Rather than the technical description, perhaps something more user-facing?
``` suggestion
    "When enabled, will remove the system names from the playlists. For example, display 'PlayStation' instead of 'Sony - PlayStation'. Changes require a restart to take effect."
```

Reply:
Hmm, I agree on the fact that the description used might be a bit too technical, however the truncation actually affects anything before the hyphen, which is not necessarily just system names. 
For example, if I create a custom playlist and rename it "Retro - Arch", the default truncation would hide "Retro - " and just display Arch, irrespective of what the text actually is.

Reply:
Only advanced users will have custom playlists like that, they'll know what they're getting themselves into :wink:

Reply:
Fair enough, you do have a point there! I updated the file with your suggested description.